corehq/apps/translations/tests/data/bulk_app_translation/download/app.json

```diff
@@ -136,8 +136,9 @@
       "case_list": {
         "doc_type": "CaseList",
         "label": {
+          "en": "Steth List"
         },
-        "show": false
+        "show": true
```

Comment:
why is this change needed?

Reply:
So that the CLMI translation shows up in the download file: https://github.com/dimagi/commcare-hq/pull/24578/files#diff-27f0b75dc11241833e40eb967bbb737aR163

Reply:
so we don't update translation if its not shown as of now, and then in future if it's made visible it would appear untranslated?
I think we should still show it and let the user decide to leave it untranslated. 
Definitely not related to this PR but just thinking if this can be improved.

Reply:
I don't think it makes sense to include this in the download if it isn't shown. Users who aren't aware of it will be confused by what it is and why it's there.

The upload doesn't look at the `show` flag, though, the user can update the translation even if it isn't being shown...but they'd have to add that row themselves (or have it in the download from an older version of the app that did show it) which probably isn't likely.

Reply:
> I don't think it makes sense to include this in the download if it isn't shown

Is there a harm in translating something even if its not shown for now? Won't the app builder be aware if they have added something like this in the app?
Agree that the whole scenario is unlikely.

Reply:
All modules have a `case_list` property, even if the user hasn't ever touched the case list menu item (or doesn't have the add-on enabled). They're even part of modules that don't use cases (this module in the test app is a surveys module, so it's illogical for it to contain a CLMI, but I was being lazy). Am I missing something?

Reply:
you definitely have more knowledge on this and i am speaking from an outside perspective with least knowledge and just user experience ..

just to confirm again
> if this is set to show false the string won't appear in the app translations download?

if thats a yes, that's where my whole point is.
If thats a no, then i got this all wrong and please ignore it.

Reply:
yes, if `show` is false, the string will not appear in the download, which I believe is desirable behavior

Reply:
Cool, thank you :)